pkg/apis/core/validation/validation_test.go

```diff
@@ -16866,6 +16885,107 @@ func TestValidateReplicationController(t *testing.T) {
 			matcher := field.ErrorMatcher{}.ByType().ByField().ByOrigin().ByDetailSubstring()
 			matcher.Test(t, tc.expectedErrs, errs)
 		})
+
+		verifyVersionedValidationEquivalence(t, &tc.input, nil)
+	}
+}
+
+func verifyVersionedValidationEquivalence(t *testing.T, obj, old k8sruntime.Object) {
```

Comment:
I mentioned in the other PR, I think, but this feels like it should run against strategy (where we are aware of versioned validation) rather than this pkg?

Reply:
Sorry, yes.  I'll incorporate.

Reply:
OK, this is now resolved. The testing is now consolidated to strategy_test (for group specific testing), and pkg/api/testing (for generic testing like fuzzing).